pytorch_blade/torch_blade/config.py

```diff
@@ -150,6 +159,10 @@ def __init__(self):
         # Also, you can pass multiple dynamic input shapes if single configure does not give satisfied
         # performance, like config.dynamic_tuning_shapes = [configure1, configure2, ...].
         self._dynamic_tuning_shapes = {}
+        # For some torchscripts exported by torch.jit.script,the format of the input may be very complex,
+        # for example, Dict. In this case, it is not easy or even possible to set tuning shapes through
+        # List. So this config allows users to directly pass inputs corresponding to min/max/opts
+        self._dynamic_tuning_inputs = {}
```

Comment:
Please update the configuration here: https://github.com/alibaba/BladeDISC/blob/main/pytorch_blade/benchmark/detectron2/run_blade.py#L274

Reply:
done